import io.netty.resolver.dns.DnsNameResolverException;
public class FunctionUse11936 {
public void funcUse() {
DnsNameResolverException dnsnameresolverexception = new DnsNameResolverException();
}
}